tests/test-cases-lib.sh

```diff
@@ -75,6 +92,45 @@ function TEST_road-traffic-predictor() {
 
 }
 
+function TEST_realtime-trade-monitor() {
+  echo "********** START TEST: realtime-trade-monitor **********"
+
+  local timeout=200
+  local pause=20
+  local demo_root=$1
+  local log_file=realtime-trade-monitor-log.txt
+
+  echo "---------- realtime-trade-monitor: preparing env - starting kafka service and creating \"trades\" topic"
+  cd $demo_root/tests/tools/qe-kafka-manager
+  timeout $timeout mvn exec:java -DkafkaTopic=trades&
+  sleep $pause
+
+  echo "---------- realtime-trade-monitor: preparing env - starting trades producer"
+  cd $demo_root/realtime-trade-monitor
+  timeout $timeout java -jar trade-producer/target/trade-producer-5.0-SNAPSHOT.jar 127.0.0.1:9092 5 | tee ${log_file}&
+  sleep $pause
+
+
+  echo "---------- realtime-trade-monitor: starting jet server"
+  timeout $timeout java -jar jet-server/target/jet-server-5.0-SNAPSHOT.jar | tee ${log_file}&
+  sleep $pause
+
+
+  echo "---------- realtime-trade-monitor: starting jet jobs to ingest and aggregate trades"
+  java -jar trade-queries/target/trade-queries-5.0-SNAPSHOT.jar load-symbols | tee ${log_file}
+  sleep $pause
+  java -jar trade-queries/target/trade-queries-5.0-SNAPSHOT.jar ingest-trades 127.0.0.1:9092 | tee ${log_file}
+  sleep $pause
+  java -jar trade-queries/target/trade-queries-5.0-SNAPSHOT.jar aggregate-query 127.0.0.1:9092 | tee ${log_file}
+  sleep $pause
+
+  echo "********** Launch data-observer to make sure landingMap is being updated by job **********"
+  cd $demo_root/tests/tools/qe-data-observer
+  timeout $timeout mvn exec:java -DmapName=query1_Results -DobserveOperation=update
```

Comment:
if we want to use `timeout` for test then we need to check whether it finished or timed out.

Try to execute following script:
```
#!/bin/sh
timeout 1 sleep 60
echo "oops, this is printed even if previous command timeouted"
```

Reply:
This is redesigned. Currently, map observer will quit in any way (client timeout has been added) so no sense to use a timeout for it. Test result is being processed separately.